pkg/services/sqlstore/user_auth.go

```diff
@@ -151,6 +151,13 @@ func GetAuthInfo(query *models.GetAuthInfoQuery) error {
 	if err != nil {
 		return err
 	}
+	if has {
+		_, err := x.Desc("created").Exec("DELETE FROM user_auth WHERE o_auth_access_token = ? AND rowid NOT IN (SELECT MIN(rowid) FROM user_auth GROUP BY o_auth_access_token)", userAuth.OAuthAccessToken)
```

Comment:
Is `rowid` something supported in sqlite? Never seen this. Does it work with mysql and postgres as well?

Reply:
Oh, I think it's not supported on Postgres (but supported in the other two) :/ 
Let me try and fix that.

Reply:
Have been thinking about the deleting of old duplicate rows and whether that's needed given we have workarounds in the issue for cleaning that up. If we just start with making sure to fix that no more duplicates are created we have fixed the biggest problem. We can decide on the delete later maybe?

Reply:
Maybe... Also can think about adding an update column by then. Try and run it again without this part then, only with this added --> https://github.com/grafana/grafana/pull/32286/files#diff-7f4243c0fd4c4d7435e078fb10c3743fb2350bb4b6091de5cfafdc3679416558R32